sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -575,7 +576,8 @@ object ScalaJSPluginInternal {
 
       loadedJSEnv <<= Def.taskDyn {
         val log = streams.value.log
-        val libs = resolvedJSDependencies.value.data
+        val libs = resolvedJSDependencies.value.data :+
+            ResolvedJSDependency.minimal(javaSysProps2JSLib(javaOptions.value))
```

Comment:
Could we avoid adding any `VirtualFile` if there turns out not to be any `-D` option? I guess `javaSysProps2JSLib` should return an `Option[VirtualJSFile]`